.storybook/webpack.config.js

```diff
@@ -1,72 +1,23 @@
 const path = require('path');
-const PackageImporter = require('node-sass-package-importer');
+const defaultConfig = require('@stoplight/storybook-config/webpack.config');
 const inliner = require('sass-inline-svg');
+const {last} = require('lodash');
 
 const cwd = process.cwd();
 
-module.exports = ({ config }) => {
-  config.context = cwd;
-  config.mode = 'development';
+module.exports = (baseConfig, env, config) => {
+  config = defaultConfig(baseConfig, env, config);
 
-  config.resolve.alias['@project/stories'] = require.resolve('src/__stories__/index.ts', { paths: [cwd] });
-  config.resolve.extensions.push('.ts', '.tsx', '.js');
-  config.resolve.modules = [...(config.resolve.modules || []), path.resolve('./')];
+  config.watchOptions = { ignored: ['dist', /node_modules\/(?!@stoplight)/] };
 
-  config.module.rules.push({
-    test: /\.(ts|tsx)$/,
-    include: [path.resolve(cwd, 'src')],
-    use: [
-      {
-        loader: require.resolve('ts-loader'),
-        options: {
-          onlyCompileBundledFiles: true, // https://github.com/TypeStrong/ts-loader#onlycompilebundledfiles-boolean-defaultfalse
-        },
-      },
-    ],
-  });
-
-  config.module.rules.push({
-    test: /\.scss$/,
-    use: [
-      {
-        loader: 'style-loader',
-        options: {
-          sourceMap: true,
-        },
-      },
-      {
-        loader: 'css-loader',
-        options: {
-          sourceMap: true,
-        },
-      },
-      {
-        loader: 'postcss-loader',
-        options: {
-          sourceMap: true,
-          ident: 'postcss',
-          plugins: loader => [require('postcss-import'), require('autoprefixer')],
-        },
-      },
-      'resolve-url-loader',
-      {
-        loader: 'sass-loader',
-        options: {
-          sourceMap: true,
-          importer: [PackageImporter()],
-          functions: {
-            'svg-icon': inliner(path.resolve('src', 'styles', 'icons'), {
-              // run through SVGO first
-              optimize: true,
-              // minimal "uri" encoding is smaller than base64
-              encodingFormat: 'uri',
-            }),
-          },
-        },
-      },
-    ],
-    include: path.resolve(__dirname, '../'),
-  });
+  // we need to override the 'svg-icon' configuration unfortunately
```

Comment:
I could have written it there, my bad.

Because the default storybook config inlines icons from `node_modules/@stoplight/ui-kit`, which understandably causes problems here.